src/styles/index.scss

```diff
@@ -1,3 +1,199 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 726px;
+  padding: 10px;
+  display: flex;
+}
```

Comment:
Properties within the `.calendar` selector should be ordered according to the checklist. Display property should come before box model properties. Please reorder them accordingly.